examples/ota-requestor-app/linux/main.cpp

```diff
@@ -117,6 +121,33 @@ OptionSet cmdLineOptions = {
 
 OptionSet * allOptions[] = { &cmdLineOptions, nullptr };
 
+// Network commissioning
+namespace {
+constexpr EndpointId kNetworkCommissioningEndpointMain      = 0;
+constexpr EndpointId kNetworkCommissioningEndpointSecondary = 0xFFFE;
+
+// This file is being used by platforms other than Linux, so we need this check to disable related features since we only
+// implemented them on linux.
+#if CHIP_DEVICE_LAYER_TARGET_LINUX
+#if CHIP_DEVICE_CONFIG_ENABLE_THREAD
+NetworkCommissioning::LinuxThreadDriver sLinuxThreadDriver;
+Clusters::NetworkCommissioning::Instance sThreadNetworkCommissioningInstance(kNetworkCommissioningEndpointMain,
+                                                                             &sLinuxThreadDriver);
+#endif
```

Comment:
Minor: Adding a comment denoting the #define flag for this endif (and others in this PR), to make it easier on the eyes...